homework-10/reservationWeekday.js

```diff
@@ -0,0 +1,42 @@
+//import halls array
+
+var halls = require ('./halls.js');
+
+
+//import luxon
+const { DateTime } = require("luxon");
+
+//an array of reserved halls - returns empty array
+
+function reserved(input, field) {
```

Comment:
Za dobivanje array-a iz drugog array-a ali samo s objektima koji zadovoljavaju određen uvjet koristi se filter funkcija. Kao i map, bude znatno preglednija od for petlje.